docs/events/timeline-ui-overview.asciidoc

```diff
@@ -178,3 +178,29 @@ From the *Correlation* tab, you can also do the following:
 * Specify the date and time range that you want to investigate.
 * Reorder the columns and choose which fields to display.
 * Choose a data view and whether to show detection alerts only.
+
+[discrete]
+[[esql-in-timeline]]
+== Use ES|QL to interact with alerts 
+
+preview::[]
+
+The {ref}/esql.html[Elasticsearch Query Language (ES|QL)] provides a powerful way to filter, transform, and analyze alert data stored in Elasticsearch. ES|QL queries use "pipes" to manipulate and transform data in a step-by-step fashion. This approach allows you to compose a series of operations, where the output of one operation becomes the input for the next, enabling complex data transformations and analysis.
+
+You can use ES|QL in Timeline by opening the **ES|QL** tab. From the tab, you can: 
+
+- Explore your alerts using the default query, or creating a custom one. The default query searches the Security alert index (`.alerts-security.alerts-default`) and  indices specified in the <<update-sec-indices,Security data view>> for alerts that were generated during the last 15 minutes. The query's output is limited to the top ten results.
```

Comment:
```suggestion
- Explore your alerts using the default query, or a custom one. The default query searches the Security alert index (`.alerts-security.alerts-default`) and  indices specified in the <<update-sec-indices,Security data view>> for alerts that were generated during the last 15 minutes. The query's output is limited to the top ten results.
```

Reply:
The top ten alerts by what? Severity? Frequency?

Reply:
I'm not actually sure how the `[LIMIT](https://security-docs_4062.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-commands.html#esql-limit)` command decides which rows/alerts count as the "top" ones. @michaelolo24 do you have any insight on this?

Reply:
Resolving this as per our discussion in Slack.